src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToGrid/Grid.java

```diff
@@ -0,0 +1,59 @@
+package edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+
+public class Grid {
+	private static Grid instance;
+	private Location selectedPos;
+	public Grid() {
+		this.selectedPos = Location.POS0;
+	}
+	
+	public static Grid getInstance() {
+		if (instance == null) {
+			instance = new Grid();
+		}
+		return instance;
+	}
+	
+	public Location getLocation(){
+		return selectedPos;
+	}
+	
+	public void setPose(Location pose){
+		 selectedPos = pose;
+	}
+	
+
+	public enum Location {
+		POS0(new Pose2d(new Translation2d(0,0),new Rotation2d())),
+		POS1(new Pose2d(new Translation2d(1,0),new Rotation2d())),
+		POS2(new Pose2d(new Translation2d(0,1),new Rotation2d())),
+		POS3(new Pose2d(new Translation2d(1,1),new Rotation2d())),
+		POS4(new Pose2d(new Translation2d(0.5,0),new Rotation2d())),
+		POS5(new Pose2d(new Translation2d(0,0.5),new Rotation2d())),
+		POS6(new Pose2d(new Translation2d(0.5,0.5),new Rotation2d())),
+		POS7(new Pose2d(new Translation2d(1,0.5),new Rotation2d())),
+		POS8(new Pose2d(new Translation2d(0.5,1),new Rotation2d()));
```

Comment:
is "pos" pose or position? name it either one, not pos (POS means piece of shit)